src/Card.js

```diff
@@ -398,6 +398,41 @@ const Card = (stack, targetElement) => {
     stack.destroyCard(card);
   };
 
+  const drag = (duration, distance, direction) => {
+    const startTime = new Date().getTime();
+
+    return $q((resolve, reject) => {
+      let finalX = distance * direction;
+
+      let interval = setInterval(_ => {
+        let progress = (new Date().getTime() - startTime)/duration;
+        let delta = Math.pow(progress, 2);
+ 
+        if(progress >= 1) {
+          clearInterval(interval);
+
+          card.throwIn(currentX, 0);
+          eventEmitter.trigger('dragend', { target: targetElement });
+          return resolve();
+        }
+        currentX = finalX * delta;
+        doMove();
+      }, 1);
```

Comment:
We should use requestAnimationFrame here. Something like: https://github.com/AyogoHealth/ayogo-client-common/blob/master/src/angular/user-directives/level-progress.ts